framework/src/main/java/org/checkerframework/framework/util/FlowExpressionParseUtil.java

```diff
@@ -158,7 +145,73 @@ private static Pattern anchored(String regex) {
     }
 
     private static boolean isMemberSelect(String s, FlowExpressionContext context) {
-        return parseMemberSelect(s) != null;
+        return parseMemberSelect(s, context) != null;
+    }
+
+    /**
+     * Matches a field access which contains a method call either in receiver or in remaining. First
+     * of returned pair is object and second is field. The object is not a field access expression
+     * itself.
+     *
+     * @param s expression string with replaced formal parameters
+     * @param origString expression string
```

Comment:
Variables `s` and `origString` are related.  They should have related names too.  The current names don't suggest any relationship between the variables.